catroid/build.gradle

```diff
@@ -106,6 +108,7 @@ dependencies {
     compile 'com.squareup.okhttp:okhttp:2.3.0'
     compile 'com.google.code.gson:gson:2.7'
     compile 'com.github.johnpersano:supertoasts:2.0@aar'
+    compile 'com.github.mrengineer13:snackbar:1.2.0'
```

Comment:
According to the [GitHub repository](https://github.com/MrEngineer13/SnackBar#deprecated), the library is deprecated, "no longer actively being developed" and suggests to use the [design support library](http://developer.android.com/tools/support-library/features.html#design).